style-guide/styles/wds-variables/_typography.scss

```diff
@@ -0,0 +1,18 @@
+// Font family - see https://www.smashingmagazine.com/2015/11/using-system-ui-fonts-practical-guide/
+$wds-font-family: 'San Francisco', 'Helvetica Neue', 'Roboto', 'Segoe UI', 'Ubuntu', 'Droid Sans', sans-serif;
+
+// Line height
+$wds-line-height: 1.5em;
+
+// Typescale sizes
+$wds-typescale-size-minus-3: 10px;
```

Comment:
The same for colors: http://wikiadesignsystem.com/#/base-styles/colors
Please make sure documentation is up to date now.